python/lib/exifedit.py

```diff
@@ -34,7 +35,7 @@ def create_mapillary_description(filename, username, email,
     heading = exif.extract_direction()
     if heading is None:
         heading = 0.0
-    heading = normalize_bearing(interpolated_heading) if interpolated_heading is not None else normalize_bearing(heading)
+    heading = normalize_bearing(interpolated_heading + offset_angle) if interpolated_heading is not None else normalize_bearing(heading + offset_angle)
```

Comment:
@bhousel as I said, sometimes in the exif from Garmin cameras the bearing is stored as hex. That's why I had that part of strange codes to try to convert hex to real bearing.  I now modify it so that we only do that hex check when we are reading bearing from EXIF (ideally, `>360` should never happen in EXIF).